app/layout.tsx

```diff
@@ -41,6 +41,27 @@ export default function RootLayout({
           name="format-detection"
           content="telephone=no, date=no, email=no, address=no"
         />
+        <title>Hyperjump Technology</title>
```

Comment:
Just removed this. Next.js will inject the metadata automatically if you’ve created it in page.tsx. More info: https://nextjs.org/docs/app/getting-started/metadata-and-og-images